.github/workflows/test-node.js.yml

```
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x]
+        node-version: [10.x, 12.x, 14.x, 15.x]
```

I think we should split this into multiple lines, like this for example
```suggestion
        node-version:
        - 10.x
        - 12.x
        - 14.x
        - 15.x
```